src/extension/wps/src/main/java/au/org/emii/wps/NetcdfOutputProcess.java

```diff
@@ -113,7 +112,7 @@ public FileRawData execute(
             encoder.prepare(new ZipFormatter(os));
             while (encoder.writeNext()) {
                 if (progressListener.isCanceled()) {
-                    throw new ProcessDismissedException(progressListener);
+                    throw new ProcessDismissedException("The job has been stopped due to excessive server resources required");
```

Comment:
We don't realky know why the job was cancelled - perhaps just "The job has been cancelled" would be more accurate?